packages/_private/angular-demo/src/demoform/demoform.component.html

```diff
@@ -0,0 +1,197 @@
+<form
+  [formGroup]="formData"
+  (reset)="reset()"
+  (ngSubmit)="submit(formData)"
+  (syn-change)="synChange($event)"
+  #form
+>
+  <!-- PersonalInformation -->
+  <demo-fieldset legend="Personal Information">
+
+    <syn-radio-group
+      formControlName="gender"
+      id="radiogroup-gender"
+      name="gender"
+      label="Please tell us your gender"
+      [required]="true"
+    >
+      <syn-radio value="f">Female</syn-radio>
+      <syn-radio value="m">Male</syn-radio>
+      <syn-radio value="other">Other</syn-radio>
+    </syn-radio-group>
+
+    <syn-select
+      formControlName="role"
+      id="select-role"
+      label="Current position"
+      name="role"
+      [required]="true"
+      v-model="formData.role"
+    >
+      <syn-optgroup label="Developers">
+        <syn-option value="backend">Backend Developer</syn-option>
+        <syn-option value="frontend">Frontend Developer</syn-option>
+      </syn-optgroup>
+      <syn-optgroup label="Other">
+        <syn-option value="lead">Team Lead</syn-option>
+        <syn-option value="other">Other (please specify in comment section below)</syn-option>
+      </syn-optgroup>
+    </syn-select>
+
+    <syn-input
+      formControlName="name"
+      id="input-text"
+      label="Name"
+      [minlength]="10"
```

Comment:
should be 5 if it should match to vanilla demo

Reply:
I initially had problems getting this to work at all using TypeScript. Adjusted :)